test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/inventory/SpecKeyFromMnemonic.java

```diff
@@ -57,16 +57,25 @@ public SpecKeyFromMnemonic linkedTo(String id) {
 	protected boolean submitOp(HapiApiSpec spec) throws Throwable {
 		byte[] seed = Bip0032.seedFrom(mnemonic);
 		byte[] privateKey = Bip0032.privateKeyFrom(seed);
+		createAndLinkSimpleKey(spec, privateKey, name, linkedId, log);
+		return false;
+	}
+
+	static void createAndLinkSimpleKey(
+			HapiApiSpec spec,
+			byte[] privateKey,
+			String name,
+			Optional<String> linkedId,
+			Logger logToUse) {
 		var params = EdDSANamedCurveTable.getByName(EdDSANamedCurveTable.ED_25519);
 		var privateKeySpec = new EdDSAPrivateKeySpec(privateKey, params);
 		var pk = new EdDSAPrivateKey(privateKeySpec);
 		var pubKeyHex = Hex.encodeHexString(pk.getAbyte());
-		log.info("Hex-encoded public key: " + pubKeyHex);
+		logToUse.info("Hex-encoded public key: " + pubKeyHex);
```

Comment:
Can't we use `log`  here ?

Reply:
Two classes are calling this `createAndLinkSimpleKey` so they pass in their respective `log`s.